nanoFirmwareFlasher/Esp32Operations.cs

```diff
@@ -284,6 +287,13 @@ internal class Esp32Operations
 
             if (updateFw)
             {
+#if FEATHER_S2_DELAY
+                if (targetName.Contains("FEATHER_S2"))
+                {
+                    Thread.Sleep(TimeSpan.FromSeconds(15));
```

Comment:
Might be worth adding an explanation as to why 15 seconds is optimal

Reply:
tried 5s and it was unreliable, noticed it fail occasionally at 10s, 15s seemed to be the minimum for guaranteed success

Reply:
Yes, I understand (saw the discord comment), but that will be lost, so better to make it a code comment 😉

Reply:
done! :-)